.eslintrc.json

```diff
@@ -1,3 +1,44 @@
 {
-  "extends": ["next/core-web-vitals", "prettier"]
+  "extends": ["next/core-web-vitals", "prettier"],
+  "env": { "es6": true },
+  "plugins": ["simple-import-sort"],
+  "rules": {
+    // Else Eslint will throw an error for `!!raw-loader!` imports
+    "import/no-webpack-loader-syntax": [0],
+    "simple-import-sort/imports": [
+      "error",
+      {
+        "groups": [
+          // Node.js builtins.
+          [
+            "^(assert|buffer|child_process|cluster|console|constants|crypto|dgram|dns|domain|events|fs|http|https|module|net|os|path|punycode|querystring|readline|repl|stream|string_decoder|sys|timers|tls|tty|url|util|vm|zlib|freelist|v8|process|async_hooks|http2|perf_hooks)(/.*|$)"
+          ],
+          // Packages. `react` related packages come first.
+          // Also, put `react-icons` in sorting order not with `react`
+          ["^react(?!-.)$", "^\\w", "^@\\w"],
+          // The Chakra theme directory if imported to story file or other places
+          ["^(\\.\\./)+@chakra-ui"],
```

Comment:
@pettinarip I would note here that doing a match with the `@/*` path alias will not work here, as TypeScript is assuming `@/@chakra-ui/theme` is a module import.

Would recommend down the road once the old `theme.ts` file is removed to consider renaming the `@chakra-ui` directory to `theme` with an `index.ts` file.

Reply:
Good point. Yes, thanks for the suggestion.

Reply:
@pettinarip when doing the fixes, I found out that TypeScript no longer throws an error about this alias. 🤔  So I went ahead and updated that as well. 😅